cmd/bnsd/Makefile

```diff
@@ -1,7 +1,8 @@
 .PHONY: all build test image tf protoc clean dist
 
 BUILD_VERSION ?= manual
-BUILD_FLAGS := -mod=readonly -ldflags "-X github.com/iov-one/weave.Version=${BUILD_VERSION}"
+APP_VERSION ?= 1
```

Comment:
As you suggested via Slack, let's make the version 101, which will correspond to `bnsd v1.0.1`, which will be rolled after this PR is merged.